packages/augur-state/src/db/AbstractDB.ts

```diff
@@ -86,5 +87,9 @@ export abstract class AbstractDB {
 
 export type PouchDBFactoryType = (dbName: string) => PouchDB.Database;
 export function PouchDBFactory(dbArgs : object) {
-  return (dbName: string) => new PouchDB(`db/${dbName}`, dbArgs);
+  const dbDir = "db";
+  if (!fs.existsSync(dbDir)){
```

Comment:
This is unnecessary if pouchdb is using the memory adapter.

Reply:
I was unable to run `yarn start` if the db directory didn't exist. I had to create the db directory, so I figured I'd just add this as a precaution.

Reply:
Submitted https://github.com/AugurProject/augur/issues/1254 to fix this later since it's an unknown amount of work, low priority, and this PR is high priority.